src/client/components/MyProfile/ProfilePage/ProfileComponent.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { handlerUserContext } from '../../../App';
+import './ProfileComponent.styles.css';
+import { ProfileCardComponent } from '../../ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../YourUploadedCVs/YourUploadedCVs';
+import TitleCvCard from '../TitleCvCard/TitleCvCard';
+
+export default function ProfileComponent() {
+  const { setUserName } = useContext(handlerUserContext);
+  const [cvsList, setCvsList] = useState([]);
+  const [userData, setUserData] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const userId = 2;
```

Comment:
To avoid hard coding ```userId``` is there any way now in this stage of developing?

Reply:
I think it needs to stay as is, as we don't have the initial login ready yet. (correct me if I'm wrong @dpfernandes)